libraries/AP_Notify/Display_SSD1306_I2C.h

```diff
@@ -10,11 +11,13 @@
 
 class Display_SSD1306_I2C: public Display {
 public:
+    Display_SSD1306_I2C();
     virtual bool hw_init();
     virtual bool hw_update();
     virtual bool set_pixel(uint16_t x, uint16_t y);
     virtual bool clear_pixel(uint16_t x, uint16_t y);
 
 private:
     uint8_t _displaybuffer[SSD1306_ROWS * SSD1306_COLUMNS_PER_PAGE];
+    AP_HAL::OwnPtr<AP_HAL::I2CDevice> _dev;
```

Comment:
I prefer to put object types before raw types (e.g. uint8_t, char, bool) in the declaration.
This comments goes for the other files as well.